source/index.html.md

```diff
@@ -5491,6 +5493,7 @@ func main() {
 |» amount|body|number|true|Amount in cents to pay the initiator|
 |» authoriser_contact_id|body|string|true|The Contact the payment will be requested from (`Contact.data.id`)'|
 |» precheck_funds|body|boolean|false|Enforce prechecking of available funds before approving the Payment Request. see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)|
+|» your_bank_account_id|body|string|false|Refers to `initiator_bank_account_id`. You can use this to identify issues related to the initiator's bank account.|
```

Comment:
In the context of an API response, this statement is correct. Here we're posting a request. (Ditto others)
```suggestion
|» your_bank_account_id|body|string|false|Refers to `initiator_bank_account_id`. You can use this to initiate a transaction with an account other than your 'Primary' bank account.|
```